droplets.go

```diff
@@ -52,6 +53,51 @@ type Droplet struct {
 	Kernel      *Kernel   `json:"kernel, omitempty"`
 }
 
+// PublicIPv4 returns the public IPv4 address for the Droplet.
+func (d *Droplet) PublicIPv4() (string, error) {
+	if d.Networks == nil {
+		return "", errors.New("no networks have been defined")
```

Comment:
var `errNoNetworks = errors.New("no networks have been defined")` to reduce duplication.